### DISCUSSION TITLE
Erratic nav bar

### DISCUSSION THREAD

**Discussion by papiermache** [requarks/wiki#6608] [Error / Bug Report] [answered]
I cannot understand what nav bar is doing - the 'Site Tree' is what we need but it makes no sense, and the Static is not useable because has no nesting.
For Site Tree - starts off as a folder icon. But when click on it - transforms to "/ (root)" and with a lower "open folder" icon indented, THEN  a line THEN a boxy icon (I assume a 'document') THEN words  "Current Directory" THEN stuff that is in that initial 'folder'

What is this all about?  Makes no sense to me, makes no sense either to the folks that are going to use this wiki.
Do I have something misconfigured? Very frustrating, and ruining my proof-of-concept demo for the company.
Any help appreciated, thx in advance. Rick

![Screenshot 2023-07-28 111448](https://github.com/requarks/wiki/assets/5385881/16dfcbe4-4fdc-4009-95f0-e148263e47a3)

**Comment by papiermache** [comment]
Additional info - as per the docs page ( https://docs.requarks.io/guide/structure ) I exepct to see a hierarchical navigational structure unfold as i drill down - but as noted above this is not what happens.

![Screenshot 2023-07-28 111856](https://github.com/requarks/wiki/assets/5385881/98b018d2-2d26-4d2f-9095-6fd09ab1c5d9)

**Comment by NGPixel** [comment, marked as answer]
When you click on a directory, you enter that directory. The files below "Current Directory" are the pages directly below that directory. The page just above it is the page with the same path as the folder (if you have one).

This may be a bit confusing but it's to allow accessing both the folder and the page of the same path.

This is being improved in v3 with proper nested menus.
